ena-submission/src/ena_deposition/create_assembly.py

```diff
@@ -586,7 +590,7 @@ def assembly_table_create(db_config: SimpleConnectionPool, config: Config, test:
         number_rows_updated = update_db_where_conditions(
```

Comment:
Rather than converting at each call site - why not do the conversion in the callee?

Reply:
Whenever you find yourself changing a lot of callers - consider modifying the callee instead